src/zkp/piaffg.rs

```diff
@@ -53,8 +53,8 @@ pub(crate) struct PiAffgProof {
 pub(crate) struct PiAffgInput {
     setup_params: ZkSetupParameters,
     g: CurvePoint,
-    N0: BigNumber,
-    N1: BigNumber,
+    pk0: PaillierEncryptionKey,
+    pk1: PaillierEncryptionKey,
```

Comment:
nit: Add documentation to these to note that they correspond to `N0`/`N1` in the text.